packages/cactus-plugin-htlc-eth-besu-erc20/src/main/json/openapi.json

```diff
@@ -0,0 +1,354 @@
+                    "id",
+                    "secret"
+                ],
+                "properties": {
+                    "id": {
+                        "description": "Contract locked id",
+                        "type": "string",
+                        "nullable": false
+                    },
+                    "secret": {
+                        "description": "Secret need to unlock the contract",
+                        "type": "string",
+                        "nullable": false
+                    }
+                }
+            },
+            "WithdrawResponse": {
+                "type": "object",
+                "properties": {
+                    "transactionReceipt": {
+                        "$ref": "#/components/schemas/Web3TransactionReceipt"
+                    },
+                    "callOutput": {}
+                }
+            },
+            "Web3TransactionReceipt": {
+                "type": "object",
+                "required": [
+                    "blockHash",
+                    "blockNumber",
+                    "transactionHash",
+                    "transactionIndex",
+                    "status",
+                    "from",
+                    "to",
+                    "gasUsed"
+                ],
+                "additionalProperties": true,
+                "properties": {
+                    "status": {
+                        "type": "boolean",
+                        "nullable": false
+                    },
+                    "transactionHash": {
+                        "type": "string",
+                        "minLength": 64,
+                        "maxLength": 64,
+                        "pattern": "/^0x([A-Fa-f0-9]{64})$/"
+                    },
+                    "transactionIndex": {
+                        "type": "number",
+                        "nullable": false
+                    },
+                    "blockHash": {
+                        "type": "string",
+                        "minLength": 64,
+                        "maxLength": 64,
+                        "pattern": "/^0x([A-Fa-f0-9]{64})$/"
+                    },
+                    "blockNumber": {
+                        "type": "number",
+                        "nullable": false
+                    },
+                    "gasUsed": {
+                        "type": "number",
+                        "nullable": false
+                    },
+                    "contractAddress": {
+                        "type": "string",
+                        "nullable": true
+                    },
+                    "from": {
+                        "type": "string",
+                        "nullable": false
+                    },
+                    "to": {
+                        "type": "string",
+                        "nullable": false
+                    }
+                }
+            }
+        },
+        "responses": {
+            "getStatusResponse": {
```

Comment:
@jordigiam Please have the schema types start with uppercase

Reply:
Done